pandunia-loge.csv

```diff
@@ -3145,6 +3157,7 @@ projete|||fra:projet, por:projeto, eng:project, spa:proyecto, rus:проект (
 Promete|name|||Prometheus|Prométhée|Prometeo|Prometeu|Прометей||||||||||||Prometheus|
 promete yum|yum 061|Pm|eng:promethium, fra:prométhium, spa:prometio, por:promécio, rus:прометий, zho:钷 (pǒ), jpn:プロメチウム, kor:프로메튬, vie:prometi, hin:प्रोमेथियम, ben:প্রমিথিয়াম, may:prometium, swa:promethi, ara: بروميثيوم|promethium|prométhium|prometio|promécio|прометий|بروميثيوم|钷|プロメチウム|프로메튬|prometi|प्रोमेथियम|প্রমিথিয়াম|prometium|promethi|prometyum|prometio|prometium|promet
 protacini yum|yum 091|Pa|eng:protactinium, fra:protactinium, spa:proactinio, por:protacnídio, rus:протактиний, zho:镤 (pú), jpn:プロトアクチニウム, kor:프로트악티늄, vie:protactini, hin:प्रोटैक्टीनियम, ben:প্রোটেক্টিনিয়াম, may:protaktinium, swa:protaktini, ara: بروتكتنيوم|protactinium|protactinium|proactinio|protacnídio|протактиний|بروتكتنيوم|镤|プロトアクチニウム|프로트악티늄|protactini|प्रोटैक्टीनियम|প্রোটেক্টিনিয়াম|protaktinium|protaktini|protaktinyum|protaktinio|protaktinium|protaktyn
+protokol|||eng:protocol, spa:por:protocolo, fra:protocole, rus:протокол (protokol), tur:may:protokol, jpn:プロトコル (purotokoru)|etiquette (protocol, procedure)|protocole|etiqueta (protocolo)|protocolo|этикет (протокол)|||礼法 (規約, プロトコル)|||शिष्टाचार||tata tertib (protokol)|itifaki|teşrifat (protokol)|||
```

Comment:
ni loga no suka mi. ye su mana si mas ka yau di has e an klar. pandunia ya ha "riti".

Reply:
oke.  ye bil si un jama loga.  tu fikir ke na tema da "buro riti"?  I think etiquette is the way you should act in formal situations with a lot of written or unwritten rules, and "buro" covers the meaning of "formal with a lot of rules".

Reply:
bon. mi bil budi la buro riti.